### DISCUSSION TITLE
No information on Copilot visible for student account, yet accepted for benefits.

### DISCUSSION THREAD

**Discussion by Iheuzio** [community/community#114082] [GitHub Education] [answered]
### Select Topic Area

Question

### Body

Hi,

I have reverified my student status on github and received the access to the benefits page, however copilot doesn't show the student free option.

![image](https://github.com/community/community/assets/97270760/55afd695-9bb7-4031-b920-b77060eafb40)

Here is the image for my Copilot tab:

![image](https://github.com/community/community/assets/97270760/2877f4f0-1052-4bec-b7b6-c86dc0fa372d)

![image](https://github.com/community/community/assets/97270760/797b4d23-5cca-4184-ba76-048a16eed406)

**Comment by michelutke** [comment, marked as answer]
Just click on Buy. In the next pop up there should be shown that you are eligible for the student version
